src/DssSpell.sol

```diff
@@ -61,140 +49,48 @@ contract DssSpellAction is DssAction {
-    address internal constant SPARK_SPELL = 0x258FD2E6b5C155aa5f3e84326A622288bd70f376;
+    address internal constant SPARK_SPELL = 0xc96420Dbe9568e2a65DD57daAD069FDEd37265fa;
 
     function actions() public override {
-        // ---------- Starknet DAI Bridge Handover ----------
-        // Forum: https://forum.makerdao.com/t/starknet-dai-handover/22033/12
-
-        // Call close on STARKNET_DAI_BRIDGE
-        StarknetDaiBridgeLike(STARKNET_DAI_BRIDGE).close();
-
-        // ---------- May 2024 AD Compensation ----------
-        // Forum: https://forum.makerdao.com/t/may-2024-aligned-delegate-compensation/24441
-        // MIP: https://mips.makerdao.com/mips/details/MIP101#2-6-3-aligned-delegate-income-and-participation-requirements
-
-        // BLUE - 41.67 MKR - 0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf
-        MKR.transfer(BLUE, 41.67 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
-
-        // Cloaky - 41.67 MKR - 0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818
-        MKR.transfer(CLOAKY, 41.67 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
-
-        // JuliaChang - 41.67 MKR - 0x252abAEe2F4f4b8D39E5F12b163eDFb7fac7AED7
-        MKR.transfer(JULIACHANG, 41.67 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
-
-        // Pipkin - 34.26 MKR - 0x0E661eFE390aE39f90a58b04CF891044e56DEDB7
-        MKR.transfer(PIPKIN, 34.26 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
-
-        // Byteron - 12.50 MKR - 0xc2982e72D060cab2387Dba96b846acb8c96EfF66
-        MKR.transfer(BYTERON, 12.50 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
-
-        // BONAPUBLICA - 11.11 MKR - 0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3
-        MKR.transfer(BONAPUBLICA, 11.11 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
-
-        // WBC - 11.11 MKR - 0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47
-        MKR.transfer(WBC, 11.11 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+        // ---------- SBE Parameter Updates ----------
+        // Forum: http://forum.makerdao.com/t/smart-burn-engine-transaction-analysis-parameter-reconfiguration-update-8/24531/
 
-        // PBG - 8.49 MKR - 0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2
-        MKR.transfer(PBG, 8.49 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+        // Decrease the hop parameter for 1,577 seconds from 11,826 seconds to 10,249 seconds.
+        DssExecLib.setValue(MCD_FLAP, "hop", 10_249);
 
-        // Rocky - 3.70 MKR - 0xC31637BDA32a0811E39456A59022D2C386cb2C85
-        MKR.transfer(ROCKY, 3.70 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+        // Decrease the bump parameter for 10,000 DAI from 75,000 DAI to 65,000 DAI.
+        DssExecLib.setValue(MCD_VOW, "bump", 65 * THOUSAND * RAD);
```

Comment:
Use `setSurplusAuctionAmount` from DssExecLib

Reply:
Amended in [0637ac4](https://github.com/makerdao/spells-mainnet/pull/414/commits/0637ac44647b0f65562432e977fdd71180ab273a)